### ISSUE TITLE
[WISHLIST] dialpad

### ISSUE THREAD

**Issue by vieridipaola** [InnovateAsterisk/Browser-Phone#24]
Hi,

I don't know if it's a "hidden" feature or if it's simply not implemented, but is it possible to have a dialpad on the main screen or a simple text input field to write a number and press enter or click a dial button? Once the SIP extension is registered, it would be easier to just dial or type a number that Asterisk then takes care of. Requiring to "add a contact" before placing a call is a show stopper when you want to either quickly dial short dynamic Asterisk extensions or call out to, say, hundreds of different destinations that you will probably only use once.

Regards,

Vieri

**Comment by vieridipaola** [comment]
I like the automatic creation of the contact upon receiving a call. All in all it's really a great project. Thanks for sharing it!
The dialer or text input field was to allow users at a hospital to contact patients on a daily basis. The phone numbers of these patients are in an EMR database. The users can view the DB through a HIS frontend (Hospital Information System) -- ie. a web frontend. The problem is that it's proprietary software, and I don't know if it's possible to set custom href values to each phone number on that system. Even if it were, would it be possible to link it somehow to Browser-Phone so that either the contact is auto-created or the call is made or both? 
In other words, is it currently possible to GET or POST or whatever a phone number to the app?

**Comment by InnovateAsterisk** [comment]
> Even if it were, would it be possible to link it somehow to Browser-Phone so that either the contact is auto-created or the call is made or both?
In other words, is it currently possible to GET or POST or whatever a phone number to the app?

It would take a mojor update to change it to a "line" based system. I'll consider it.

**Comment by InnovateAsterisk** [comment]
There is now one. (next to "Add Someone" icon)

![image](https://user-images.githubusercontent.com/58937573/85033672-81838880-b181-11ea-988a-ef1a013c9c73.png)
